src/libasr/codegen/asr_to_c.cpp

```diff
@@ -490,6 +490,20 @@ R"(
         array_types_decls += open_struct + body + end_struct;
     }
 
+    void visit_ComplexConstant(const ASR::ComplexConstant_t &x) {
+        headers.insert("complex");
+
+        std::string re = std::to_string(x.m_re);
+        std::string im = std::to_string(x.m_im);
+
+        std::string type_name = "float complex";
+        // if (x.m_type == 8) type_name = "double complex";
+
+        src = "(" + type_name + ") (" + re + ", " + im + ")";
+
+        last_expr_precedence = 2;
```

Comment:
Its fine to have it as the same pattern is followed in other constant nodes (IntegerConstant, LogicalConstant, etc.).

Reply:
nice! thanks!